arwen/host/arwen.go

```diff
@@ -45,10 +46,10 @@ type vmHost struct {
 // NewArwenVM creates a new Arwen vmHost
 func NewArwenVM(
 	blockChainHook vmcommon.BlockchainHook,
-	cryptoHook vmcommon.CryptoHook,
 	hostParameters *arwen.VMHostParameters,
 ) (*vmHost, error) {
 
+	cryptoHook := crypto.NewCryptoVm()
```

Comment:
Suggest to rename the constructor `NewCryptoVm()` to `NewVMCrypto()`.

Reply:
Done